test/easyconfigs/easyconfigs.py

```diff
@@ -926,6 +927,31 @@ def test_easyconfig_locations(self):
                     if not (dirpath.endswith('/easybuild/easyconfigs') and filenames == ['TEMPLATE.eb']):
                         self.assertTrue(False, "List of easyconfig files in %s is empty: %s" % (dirpath, filenames))
 
+    def test_easyconfig_name_clashes(self):
+        """Make sure there is not a name clash when all names are lowercase"""
+        topdir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+        names = defaultdict(list)
+        for (dirpath, _, _) in os.walk(topdir):
+            # ignore git/svn dirs & archived easyconfigs
+            if '/.git/' in dirpath or '/.svn/' in dirpath or '__archive__' in dirpath:
+                continue
+            dirpath_split = dirpath.replace(topdir, '').split(os.sep)
+            if len(dirpath_split) == 5:
+                name = dirpath_split[4]
+                names[name.lower()].append(name)
```

Comment:
Can we use `robot_path` instead of jumping 3 levels up?  It would be even better if we could use a case insensitive version of `robot_find_easyconfig` from framework, then we can benefit from the indexes.

Reply:
The jump three levels up was copied from the test above this one. I am unsure what might happen is we use `robot_path` when running the tests locally if we have that set.

For `python3` (RHEL 8.6 default version) the speed is not an issue:
```
real    0m0.321s
user    0m0.108s
sys     0m0.184s
```

Sadly, for `python2` it is a lot slower:
```
real    0m49.083s
user    0m0.588s
sys     0m1.439s
```